```
Unformatted:
  src/runtime/interrupt/interrupt_gameboyadvance.go
make: *** [GNUmakefile:213: fmt-check] Error 1
```

Please run `go fmt` so this PR can pass the CI tests. Thanks!

--- a/src/runtime/interrupt/interrupt_gameboyadvance.go
+++ b/src/runtime/interrupt/interrupt_gameboyadvance.go
@@ -23,7 +23,7 @@ func handleInterrupt() {
 	for i := 0; i < 14; i++ {
 		if flags&(1<<uint(i)) != 0 {
 			gba.INTERRUPT.IF.Set(1 << uint(i)) // acknowledge interrupt
-			gba.BIOS_IF.Set(1 << uint(i)) // acknowledge interrupt for BIOS
+			gba.BIOS_IF.Set(1 << uint(i))      // acknowledge interrupt for BIOS
 			callInterruptHandler(i)
 		}
 	}

